src/main/java/com/autotune/analyzer/services/UpdateResults.java

```diff
@@ -59,10 +59,14 @@ public class UpdateResults extends HttpServlet {
     private static final long serialVersionUID = 1L;
     private static final Logger LOGGER = LoggerFactory.getLogger(UpdateResults.class);
     public static ConcurrentHashMap<String, PerformanceProfile> performanceProfilesMap = new ConcurrentHashMap<>();
+    public static ConcurrentHashMap<String, KruizeObject> mainKruizeExperimentMAP;
 
     @Override
     public void init(ServletConfig config) throws ServletException {
         super.init(config);
+        mainKruizeExperimentMAP = (ConcurrentHashMap<String, KruizeObject>) config.getServletContext().getAttribute(AnalyzerConstants.EXPERIMENT_MAP);
+        if (mainKruizeExperimentMAP == null)
+            mainKruizeExperimentMAP = new ConcurrentHashMap<>();
```

Comment:
What is the scope of this object?  Why is this not part of `doPost`

Reply:
removed